### DISCUSSION TITLE
is this ready to replace ExpressJS ?

### DISCUSSION THREAD

**Discussion by ceddybi** [uNetworking/uWebSockets.js#81] [Q&A] [answered]
Whole alien typer shit you made here 🙌🙌🙏👌

Am planning to replace my expressjs servers with this to power my graphql services.

Do you have any plans of mimicking expressjs's api, am very sure most people will ditch express for this.

Like making their migrations seamlessly without breaking their legacy setups/routes.

**Comment by ghost** [comment]
No that is not the purpose. This project is aimed towards companies who are willing to put some effort into solving their business problems.

This project does not compare with Express, it compares with Nodejs itself. You are essentially moving a large part away from Nodejs by using this code.

**Comment by dalisoft** [comment]
@ceddybi I think yes, can you look to [my library](https://github.com/dalisoft/nanoexpress)?

**Comment by ceddybi** [comment, marked as answer]
Thank you @dalisoft , am now using your lib, super sweet

**Comment by dalisoft** [comment]
Thank you @ceddybi for using my library.
Happy new year
